airbyte-webapp/src/utils/crossDomainUtils.ts

```diff
@@ -0,0 +1,12 @@
+export const setSegmentAnonymousId = (queryString?: string): void => {
+  const queryParams = new URLSearchParams(queryString);
+  const ajs_anonymous_id = queryParams.get("ajs_anonymous_id");
+
+  if (ajs_anonymous_id) {
+    localStorage.setItem("ajs_anonymous_id", JSON.stringify(ajs_anonymous_id));
```

Comment:
Given that we only want to store a string here, I think we should get rid of the `JSON.stringify` (and `JSON.parse` above), which just add unnecessary performance overhead and making the types less safe (since `JSON.parse` does not know what the result type will be).

Reply:
It wasn't working correctly without `stringify` + `parse`